javascript/chronometer.js

```diff
@@ -1,26 +1,58 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0
   }
   startClick(callback) {
+    this.intervalId = setInterval(
+      () => {
+        this.currentTime++;
+            if (callback) {
+                callback()
+            }
+    }, 1000)
+    //console.log("this.startClick")
+}
+
+    
     // ... your code goes here
-  }
+    
+  
+  //setInterval = (startClick() => {),1000}
   getMinutes() {
     // ... your code goes here
+    const minutes = Math.floor(this.currentTime / 60);
+    return minutes
+
   }
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 60);
```

Comment:
There's no need to use `Math.floor()` here as the remainder will always be a whole integer
```
return this.currentTime % 60;